cmd/schedule-builder/README.md

```diff
@@ -0,0 +1,119 @@
+# Schedule Builder
+
+This simple tool have the objective parsing the schedule yaml file located in [SIG-Release](https://github.com/kubernetes/sig-release/blob/master/releases/) where shows in machine readable format the scheduled releases and past patch releases of the current Kubernetes Releases.
+
+## Install
+
+The simplest way to install the `schedule-builder` CLI is via `go get`:
+
+```
+GO111MODULE=on go get k8s.io/release/cmd/schedule-builder
```

Comment:
I think it works also without setting the env var:

```suggestion
$ go get k8s.io/release/cmd/schedule-builder
```

Reply:
done